src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -692,15 +692,15 @@ Chttp2ServerListener::~Chttp2ServerListener() {
 void Chttp2ServerListener::Start(
     Server* /*server*/, const std::vector<grpc_pollset*>* /* pollsets */) {
   if (server_->config_fetcher() != nullptr) {
```

Comment:
We seem to be tying too many things together here. For this change I'd expect no changes to chttp2 to be necessary. Suggest undoing this dependency prior to making this change.